src/main/java/com/aiykr/iquais/service/UserService.java

```diff
@@ -3,13 +3,13 @@
 import com.aiykr.iquais.dto.request.PostUserDTO;
 import com.aiykr.iquais.dto.response.Response;
 import com.aiykr.iquais.dto.response.UserResponseDTO;
-import com.aiykr.iquais.exception.ResponseExceptionHandler;
+import com.aiykr.iquais.exception.IquaisException;
 import org.springframework.http.ResponseEntity;
 
 import java.util.List;
 
 public interface UserService {
-    public ResponseEntity<Response> createUser(PostUserDTO postUserDTO) throws ResponseExceptionHandler;
+    public ResponseEntity<Response> createUser(PostUserDTO postUserDTO) throws IquaisException;
```

Comment:
Arent methods declared in interface by default public?

Reply:
removed